src/mesa/drivers/dri/i965/brw_draw_upload.c

```diff
@@ -631,6 +631,8 @@ brw_prepare_vertices(struct brw_context *brw)
 
       buffer->bo = intel_bufferobj_buffer(brw, enabled_buffer[i], start,
                                           range, false);
+      if (buffer->bo == NULL)
```

Comment:
There is something completely wrong somewhere else if we end up in this situation, please remove this.